scoda/static/src/sass/components/_cta_block.scss

```diff
@@ -167,24 +172,80 @@
         } 
             
     }
+    .cta-field-description{
+        &-list{
+            list-style: none;
+            display: flex;
+            margin-left: 20px;
+            li{
+                display: inline;
+              
+            }
+         &-detailed{
+             margin-left: 41px;
+             margin-top:20px !important;
+             margin-bottom:20px ;
+             margin-right: 20px;
+             width: 87%;
+             margin: 0 auto;
+             &-btn{
+                 margin-top:80px !important;
+             }
+         }
+        }
+
+    }
+    .cta-search{
+        height: 403px ;
+    }
+
     // Mobile specific styling
     @media (max-width: 768px) {
+
+        .cta-search{
+            position: relative;
+            margin-top: 28px !important;
+            margin-left: -10px !important;
+        }
+      
+        .cta-field-description{
+            margin-left: 20px;
+            &-list{
+                list-style: none;
+                display: block;
+                
+                
+             
+             &-detailed{
+                
+                 margin-top:20px;
+                 margin-bottom:20px ;
+                 &-btn{
+                     margin-top:80px !important;
+                 }
+             }
+            }
+    
+        }
         .cta-container{
             width: 90%;
+            margin-left: 20px !important;
         }
         &.cta_block_insight{
             .cta-container{
                 padding-bottom: 50px;
             }
         }
         .cta-info{
+            
             .cta-image{
                 width: 100% !important;
                 background-size: 100% 100%;
                 margin-bottom: 30px;
+                margin-left:0px;
```

Comment:
Please use best practise for spacing declarations, properties and nesting:

```
.cta {
 &-image {
   margin-left: 0;
  }
 &-info { ... }
}
```
Note that 0px is not used, just 0. There should be space after the property like `margin-left: 0`, not `margin-left:0;`

Reply:
@PhakamaniXulu